tests/DoctrineIntegration/ORM/phpstan.neon

```diff
@@ -0,0 +1,6 @@
+includes:
+	- ../../../extension.neon
+
+parameters:
+	doctrine:
+		objectManagerLoader: tests/DoctrineIntegration/ORM/entity-manager.php
```

Comment:
I much prefer how in #18 (or #47 :) ) i can just specify the mappings here, this basically makes a whole bunch of code that has to be copy pasted into any project using this - surely this file could be actually part of the extension?

Reply:
With #18 or #47, you always need to copy the mappings here.

With this approach, you just need to provide a file with your real EntityManager which is simple enough in DI-driven application.

Reply:
I guess that makes sense.